protected/components/OEPatientEventDateValidator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * OpenEyes.
+ *
+ * Copyright OpenEyes Foundation, 2017
```

Comment:
This raises an interesting question: are we breaking the copyright by copying their copyright?